articles/overview/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Introduction to quantum computing
+description: Learn what quantum computing is, what quantum computers can do, and how you can learn quantum computing.
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.introduction
+---
+
+# Introduction to quantum computing 
+
+There are some problems so difficult, so incredibly vast, that even if every supercomputer in the world worked on the problem, it would still take longer than the lifetime of the universe to solve.
+
+Quantum computers hold the promise to solve some of our planet's biggest challenges - in environment, agriculture, health, energy, climate, materials science, and problems not yet even imagined. The impact of quantum computers will be far-reaching and have as great an impact as the creation of the transistor in 1947, which paved the way for today’s digital economy.
+
+## What is quantum computing?
+
+In modern usage, the word quantum means the _smallest possible discrete unit of any physical property_, usually referring to atomic or sub-atomic particles. Quantum computers use actual quantum particles – atoms or ions – as processing units, and are large, complex, and expensive devices. 
```

Comment:
Superconducting qubits and topological qubits tend are more macroscopic than individual quanta. Even atoms and ions are not the individual quanta, but are comprised of many subatomic particles, each of which themselves are composed of quarks, gluons, etc.